proto/umee/oracle/v1/query.proto

```diff
@@ -252,6 +252,8 @@ message QueryMedians {
   option (gogoproto.goproto_getters) = false;
   // denom defines the denomination to query for.
   string denom = 1;
+  // numStamps defines the number of median stamps to query for.
```

Comment:
Let's define what happens if `numStamp > max amount of stamps`. I suggest that we will handle it as the max.

Reply:
Also, add a comment that must be bigger than zero.

Reply:
Handle that case as max and added the comment👍